docs/mesos/UsageGuideUnifiedContainerizer.md

```diff
@@ -0,0 +1,121 @@
+<!--- master only -->
+> ![warning](../images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.18.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Using Calico Mesos and the Unified Containerizer
+The following information includes application json and information on launching tasks in a Mesos Cluster with Calico.
+
+## Prerequisites
+This guide assumes you have a cluster configured with a Mesos Master and at least
+one Mesos Agent running Calico Mesos.
+
+#### Vagrant install
+You can easily set up a cluster with the base requirements by following our
+[Vagrant guide](./Vagrant.md).
+
+#### Manual install
+You can install Calico Mesos on an agent by following the instructions in our
+[Manual Install Calico Mesos guide](ManualInstallCalicoMesos.md).
+
+All of the following services are required and can be installed by following
+the instruction in our [Mesos Cluster Preparation guide](MesosClusterPreparation.md):
+ - On Master
+    - Mesos Master
+ - On Agent
+    - Mesos Slave
+    - Netmodules
+ - Anywhere
+    - Marathon
+    - Zookeeper
+    - Etcd
+
+## Launching Tasks with Marathon
+Calico is compatible with all frameworks which use the new NetworkInfo protobuf when
+launching tasks. Marathon has introduced limited support for this in v0.14.0.
+
+### Launching Marathon
+If you have not installed Marathon, you can run Marathon directly by running
+the commands [here](MesosClusterPreparation.md#marathon), or you can quickly
+start it as a Docker container like the following:
+
+```
+docker run \
+-e MARATHON_MASTER=zk://<ZOOKEEPER_IP>:2181/mesos \
+-e MARATHON_ZK=zk://<ZOOKEEPER_IP>:2181/marathon \
+-p 8080:8080 \
+mesosphere/marathon:v0.14.0
+```
+
+### Launching Tasks
+Marathon-v0.14.0 supports two new fields in an application's JSON file:
+
+- `ipAddress`: Specifiying this field grants the application an IP Address
+networked by Calico.
+- `groups`: Groups are roughly equivalent to Calico Profiles. The default
+implementation isolates applications so they can only communicate with
+other applications in the same group. Assign the static `public` group
+to a task to allow it to communicate with any other application.
+ 
+> See [Marathon's IP-Per-Task documentation][marathon-ip-per-task-doc] for more information.
+
+The Marathon UI does not yet include a field for specifiying NetworkInfo,
+so we'll use the command line to launch an app with Marathon's REST API.
+
+#### Create a JSON file
+
+Below is a sample JSON application that is configured to receive an address
+from Calico. Copy this into a file called `app.json`:
+
+```
+{
+    "id": "hello-world-1",
+    "cmd": "ip addr && sleep 30",
```

Comment:
Lets change this to be `&& sleep 1000000`. In general marathon tasks are usually persistant tasks that don't exit often